src/Sulu/Component/Media/SystemCollections/UnrecognizedSystemCollection.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Media\SystemCollections;
+
+/**
+ * Indicates not existing system collection.
+ */
+class UnrecognizedSystemCollection extends \Exception
+{
+    /**
+     * @var string
+     */
+    private $key;
+
+    /**
+     * @var string []
+     */
+    private $recognizedSystemCollections;
+
+    public function __construct($key, array $recognizedSystemCollections)
+    {
+        parent::__construct(sprintf('Unrecognized system collection "%s"', $key));
```

Comment:
i have thought about this but i think the amount of system collections will be a lot so i think we should not add this into the messages